src/js/components/Image.js

```diff
@@ -18,12 +20,32 @@ export default class Image extends Component {
       }
     );
 
-    return <img id={this.props.id} className={classes} src={this.props.src} />;
+    let figcaption;
+    if (caption) {
+      figcaption = (
+        <figcaption className={`${CLASS_ROOT}__caption`}>
+          <Label>{caption}</Label>
+        </figcaption>
+      );
+    }
+    return (
+      <figure className={classes}>
```

Comment:
Should we only add the `<figure>` when there is a caption? It would keep the DOM cleaner.